finders.go

```diff
@@ -256,8 +256,32 @@ func (q *Query) eagerAssociations(model interface{}) error {
 //
 // 	q.Where("name = ?", "mark").Exists(&User{})
 func (q *Query) Exists(model interface{}) (bool, error) {
-	i, err := q.Count(model)
-	return i != 0, err
+	tmpQuery := Q(q.Connection)
+	q.Clone(tmpQuery) //avoid mendling with original query
```

Comment:
typo, should be   "avoid meddling with the original query"

Reply:
That's a copy-paste from `Count`, I'll fix it there too. :)